### ISSUE TITLE
Setup continuous integration for the client C# project

### ISSUE THREAD

**Issue by naitoarekkus** [P7-Team/P7-Client#38]
**Is your feature request related to a problem? Please describe.**
It is related to issue #37 , in that issue 37 needs to be finished before start.

**Describe the solution you'd like**
A continuous integration workflow should be setup for the client C# project.
This should automatically run all automatic tests and check that they all pass, before they will allow a pull request to be merged into the main/develop branch.

**Describe alternatives you've considered**
Github actions could be a way to set it up

**Additional context**
N/A
